kivy/graphics/vertex_instructions_line.pxi

```diff
@@ -174,7 +174,7 @@ cdef class Line(VertexInstruction):
             self.prebuild_rounded_rectangle()
         elif self._mode == LINE_MODE_BEZIER:
             self.prebuild_bezier()
-        if self._width == 1.0:
+        if self._width <= 1:
```

Comment:
yeah the difference is that 1.0 != 1:
which means that 90% of kivy users will have rendering issues (because of how the new line processes corners) on lines <= 1. which means that if you have little segments, either bar onse self computation or formula error, you will get an infinite corner that extend beyond your viewport.

Reply:
The other 10% is divided by luck or because they know what they are using exactly.

Reply:
This greatly affects on how we render rounded corners, specially because if the user uses more than available pixels, we would have corners that are far too close to render property with the new render instruction. and, to be honest, the old render instruction works perfectly, since we can't really have slimmer lines on a fixed grid and in most devices, anything less than 1 px will be either ommited by the screen or the user will not be able to see it properly. because of the density of the device it will be shown.

Reply:
all tho, it is really interesing the fat that a line width is not the absolute width but rather the double of settled size
i don't think is intituive at all :0